draft-ietf-idr-5g-edge-service-metadata-24.xml

```diff
@@ -321,16 +328,23 @@
 
         <section title=" Metadata Path Attribute Processing">
           <t>A BGP speaker that advertises a path received from one of its
-          neighbors SHOULD advertise the BGP Metadata Path attribute received
+          neighbors SHOULD advertise the BGP Metadata Path Attribute received
           with the path without modification as long as the BGP Metadata Path
-          attribute was acceptable. If the path did not come with a BGP
-          Metadata Path attribute, the speaker MAY attach a BGP Metadata
+          Attribute was acceptable. If the path did not come with a BGP
+          Metadata Path Attribute, the speaker MAY attach a BGP Metadata
           Attribute to the path if configured to do so.</t>
+	  <!-- XXX JMH - this is a place where the procedures need to be
+	  subject to the Limited Domain principle.  How do you want to discuss
+	  that check here? -->
 
-          <t>A BGP Peer receiving a BGP Metadata Path attribute should ignore
+          <t>A BGP Peer receiving a BGP Metadata Path Attribute should ignore
           Sub-TLVs with unknown types and process the recognized Sub-TLVs. BGP
           Peers should not delete any Sub-TLV from the BGP Metadata Path
           Attribute.</t>
+	  <!-- XXX JMH - The issue with inconsistently ignoring features that
+	  impact path selection across and AS is you can end up with forwarding
+	  loops.  Discussing this consistency needs to be at least broached
+	  here. -->
```

Comment:
the proposed Metadata Path Attribute is for iBGP. Why ignoring the unrecognized Sub-TLVs can cause forwarding loops? 

Does change to the following wording help? 

_"To prevent forwarding loops and ensure consistent routing decisions, it is essential that all BGP peers within an autonomous system (AS) adopt a unified approach to handling BGP Metadata Path attributes. Specifically, BGP peers should consistently ignore Sub-TLVs with unknown types while processing the recognized Sub-TLVs. Additionally, BGP peers should refrain from deleting any Sub-TLV from the BGP Metadata Path attribute. This ensures that all peers have a common understanding of the routing information and reduces the risk of routing inconsistencies that could lead to forwarding loops."_